test-packages/in-repo-addon-test.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
```

Comment:
I find it somewhat odd to have this file "loose" in the same directory as other packages.